<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QResultExprWidget_Error</class>
 <widget class="QWidget" name="QResultExprWidget_Error">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>95</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>95</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnClose">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(200, 0, 0);</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QResultExprWidgetButtonLine" name="m_editErrorResult">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QResultExprWidgetExprComboBox" name="m_cbbExpr">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>误差限</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QResultExprWidgetButtonLine" name="m_editErrorValue">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>误差结果</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QResultExprWidgetButtonLine" name="m_editResultParas">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>结果参数</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="m_btnResultParasDel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="m_btnErrorResultDel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="m_cbbExprType">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>30</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>绝对误差</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>相对误差</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QPushButton" name="m_btnErrorValueDel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QResultExprWidgetButtonLine" name="m_editStdParas">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="m_btnStdParasDel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>基准参数</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QResultExprWidgetExprComboBox</class>
   <extends>QComboBox</extends>
   <header>ResultExprWidgetItem.h</header>
  </customwidget>
  <customwidget>
   <class>QResultExprWidgetButtonLine</class>
   <extends>QLineEdit</extends>
   <header>ResultExprWidgetItem.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
